cocos/video/video-player.ts

```diff
@@ -390,7 +384,15 @@ export class VideoPlayer extends Component {
         this._impl.syncStayOnBottom(this._stayOnBottom);
         this._impl.syncKeepAspectRatio(this._keepAspectRatio);
         this._impl.syncFullScreenOnAwake(this._fullScreenOnAwake);
```

Comment:
those video source related status all need to sync when change new video source. Otherwise, those status will be defualt.